datafusion/core/src/physical_plan/file_format/mod.rs

```diff
@@ -401,6 +404,33 @@ fn create_dict_array(
     ))
 }
 
+/// A single file or part of a file that should be read, along with its schema, statistics
```

Comment:
Maybe I am missing something but I don't see "schema" and "statistics" on this struct

Perhaps we should refer to "extensions" instead?

Reply:
This description has been copied from PartitionedFile, at first I had similar impression that something is wrong, but after second read actually It doesn't tell that schema and statistics are part of the struct, but rather that these should be read from file along with data